.travis.yml

```diff
@@ -18,12 +18,26 @@ env:
   - CARGO_FLAGS="--features 'strict'"
 
 before_script:
-  # Install rustfmt 0.9.0 if it isn't already installed on
-  # Travis. This can be slow to install, so raise the Travis timeout.
-  - (which rustfmt && rustfmt --version && [[ "$(rustfmt --version)" =~ "0.9.0" ]]) || travis_wait cargo install --force rustfmt --vers 0.9.0
+  - if [ $TRAVIS_OS_NAME == linux ]; then ./autogen.sh && ./configure; fi
+  # TODO: add the necessary dependencies so we can use the same configure flags on OS X.
+  - if [ $TRAVIS_OS_NAME == osx ]; then ./autogen.sh && ./configure --without-makeinfo --with-xpm=no --with-gif=no --with-gnutls=no; fi
 
 script:
-  - ./.travis.sh
+  - make -j 3 && echo '==> Running tests ' && make test
```

Comment:
That must be `make check`, else it doesn't run anything :)

(Also, trailing space?)